### DISCUSSION TITLE
Reconnecting the stream objervation

### DISCUSSION THREAD

**Discussion by rpatidar** [bluenviron/mediamtx#1625] [Questions]
1. When the stream is disconnected and connected back WEBRTC many of the times does not work e.g. on the URL http://localhost:8889/mystream/ .
2. If i open a new tab with the HLS Stream on http://localhost:8888/mystream/, interestingly the WEB RTC starts to work at the same time in 1-2 seconds after that. 

i can record the video and share as well.

**Comment by aler9** [comment]
Hello, this is a browser issue: when a browser detects that a WebSocket connection is failing too many times, it temporarily halts new WebSocket connections. If you open an additional tab and everything goes right, the lock on WebSocket is removed and connectivity is restored on the WebRTC page.

Anyway, if you want to report server issues, use Issues and not Discussions.

[a comment by an automated account is not shown]
